### ISSUE TITLE
"Accessibility" button is automatically selected as an answer and cannot be unselected

### ISSUE THREAD

**Issue by margaretmariano** [7-11-22-sdet-hooper/ReviewsFrontend#12]
####Steps to reproduce
1. Navigate to https://sdetinterview.com/
2. Click on Quiz 
3. Click on multiple choice answers and Click Next until you reach the question “This testing is generally done after a change, fix or feature addition.”
4. Click “Accessibility”
5. Click Next to question “Test to validate specific behaviors in a focused area within a system.”

####Current behaviour (bug)
1. “Accessibility” button is automatically selected
![accessibilitybug](https://user-images.githubusercontent.com/54867999/186696073-08bf7861-d063-4bc8-9c13-7affafeaf339.png)
. 
2. “Accessibility” button is unable to be unselected.

####Expected behaviour (correct)
All multiple choice buttons are not selected yet

####Relevant logs and/or media (optional)
Write here or upload an image/video

####Other comment (optional)
Write here

####Reported by
Margaret
